doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -288,11 +289,20 @@ Check details in the document about [initial notifications](initial_notification
 
 Apart from the `status` values defined for subscription in the NGSIv2 specification, Orion also allows to use `oneshot`. Please find details in [the oneshot subscription document](oneshot_subscription.md)
 
-TBD: to be completed by Fermin
+## Notify only attributes that change
+
+Orion supports an extra field `onChangedAttrs` (within `notification`) in subscriptions, apart of the ones described in
```

Comment:
Hi @fgalan Sir,

There is only one change is required from my side, we have introduced a new field name as ```onlyChangedAttrs```, not ```onChangedAttrs```.  Could you please replace it to ```onlyChangedAttrs``` at all the places.

Thanks

Reply:
Good point!

Fixed in https://github.com/telefonicaid/fiware-orion/pull/3528/commits/4f3db77288420976bf39dc376c8b3b1e6e52f701

Reply:
Thank you @fgalan Sir, for updation in the document.